File: sdklibrary/src/main/java/com/sdk/sdklibrary/base/BasePresenter.java
package com.sdk.sdklibrary.base;


/**
 * <p>Description:
 * @author colin
 * Date:2023-02-08
 * 绑定BaseView
 * 解绑BaseView
 */

public interface BasePresenter<T extends BaseView> {

    void attachView(T t);
    void detachView();
}

File: sdklibrary_kt/src/main/java/com/sdk/sdklibrary/base/BasePresenter.kt
package com.sdk.sdklibrary.base

/**
Data:2023/2/21
Time:19:07
author:colin handsome
 * 绑定BaseView
 * 解绑BaseView
 */
open interface BasePresenter<T : BaseView?> {
    fun attachView(t: T)
    fun detachView()
}